### PR TITLE
fix: Rehaul SAML SSO landing page and Auth0

### PR DESCRIPTION
**Pull request by Pranay105**
- Updated the landing page
- Refined the content so that it's easier to follow
- Updated images

### CODE REVIEW

File: website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    c. `<X509Certificate>`
+    
+    d. `<NameIDFormat>`
+
+2. In the Appsmith **SAML 2.0** configuration page, click **IdP Data** under the **Register Identity Provider** section. Add the following values from XML tags as per the table below:
+
+| <div style= {{width:"200px"}}> **Appsmith Field ** </div>  | <div style= {{width:"auto"}}> **Metadata XML Tag**  </div>                                                                      |
+| ------------------- | ------------------------------------------------------------------------------------------ |
+| **Entity ID**               | Add the value of **entityID** attribute from `<EntityDescriptor>` tag.                 |
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx:66]:
```suggestion
| **Entity ID**               | Paste the value of the **entityID** attribute specified in the `<EntityDescriptor>` tag.                 |
```

**Review by danciaclara — CHANGES_REQUESTED** [review]

File: website/static/img/Azure-Active-Directory.png
[the remark this answers is not in the document]
Comment (danciaclara) [reply website/static/img/Azure-Active-Directory.png:1]: Screenshot 2023-05-11 at 20 51 24
the ones in the center look bigger

File: website/static/img/Autho_create-app.png
Comment (danciaclara) [inline website/static/img/Autho_create-app.png:1]: remove this image if not used

File: website/static/img/auth0_SAML_metadata-url.png
Comment (danciaclara) [inline website/static/img/auth0_SAML_metadata-url.png:1]: remove this image if not used

**Review by danciaclara — CHANGES_REQUESTED** [review]

File: website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/README.md [hunk 2]
```diff
@@ -1,139 +1,53 @@
...
+                <a href="/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/active-directory">
+                <img class="containerImage" src="/img/Azure-Active-Directory.png" alt="AD-logo"/>
+                </a>     
+            </div> 
+            <b><a href="/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/active-directory">Active Directory</a></b><br/>
+        </div>
+        <div class="columnGrid column-one" align="center">
+          <div class="containerCol">
+                <a href="/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/okta">
+                <img class="containerImage" src="/img/okta.png" alt="okta-logo"/>
```
[the remark this answers is not in the document]
Comment (danciaclara) [reply website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/README.md]: ignore. Looks like linter is fine now. @Pranay105 did you do anything to change it?

File: website/static/img/Azure-Active-Directory.png
[the remark this answers is not in the document]
Comment (Pranay105) [reply website/static/img/Azure-Active-Directory.png:1]: All the images are of equal size and don't have any background. If I change the size, the text alignment breaks.

File: website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+# Auth0
+
+To configure Appsmith to use [Auth0](https://auth0.com/) as a SAML provider, follow the steps below:
+
+## Prerequisites
+
+1. In Appsmith, go to **Admin Settings** > **Authentication** and click **Enable** on  **SAML 2.0**.
+
+2. Copy the **Redirect URL** and **Entity URL** from the **SAML 2.0** configuration page to add them later in the Auth0 settings.
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx:17]: is the Entity URL getting pasted anywhere?

**Review by danciaclara — CHANGES_REQUESTED** [review]

File: website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx (hunk 3 above)
Comment (Pranay105) [reply website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx:17]: No it's not in Auth0.

File: website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+##  Configure Auth0
+
+1. Log in to your [Auth0](https://auth0.com/) account and go to **Applications** > **Create Application**. 
+2. In the **Create application** modal, select **Regular Web Application** and click **Create**.
+3. Once your application is created, you’re taken to the Quick Start screen. Go to the **Settings** tab.
+4. Scroll down to the **Application URIs** and add the **Redirect URL** in the **Allowed Callback URLs** field.
+5. Click **Save Changes**.
+
+## Configure Auth0 in Appsmith
```
[the remark this answers is not in the document]
Comment (danciaclara) [reply website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx]: using the same language as on the platform

File: website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Pranay105) [reply website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/README.md]: Yes. I changed to Okta in "alt".

File: website/static/img/Azure-Active-Directory.png
[the remark this answers is not in the document]
Comment (danciaclara) [reply website/static/img/Azure-Active-Directory.png:1]: Screenshot 2023-05-11 at 19 19 25
Screenshot 2023-05-11 at 19 19 28

check the empty area around Auth0 logo

File: website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx (hunk 1 above)
Comment (Pranay105) [reply website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx]: Sorry I committed it. But the change you suggested here changes the meaning of the sentence. You have to paste the value of the entityID attribute present in the `<EntityDescriptor>` tag to the Entity ID in Appsmith. I'm telling the user what to add in Entity ID.

File: website/static/img/Azure-Active-Directory.png
[the remark this answers is not in the document]
Comment (danciaclara) [reply website/static/img/Azure-Active-Directory.png:1]: Added the fix for the images

**Review by danciaclara — APPROVED** [review]

**Pull request merged**